### DISCUSSION TITLE
Curation report Nov 28-Dec4

### DISCUSSION THREAD

**Discussion by susancoort** [wikipathways/GPML-Repo#105] [Curation reports]
-  Updated the approved and Lipid maps tags for [WP4346](https://www.wikipathways.org/index.php/Pathway:WP4346)
-  Under-construction pathways:
        - @DeniseSl22 Can I remove the under-construction tags for [WP5276](https://www.wikipathways.org/index.php/Pathway:WP5276), [WP5277](https://www.wikipathways.org/index.php/Pathway:WP5277), [WP5279](https://www.wikipathways.org/index.php/Pathway:WP5279), [WP5280](https://www.wikipathways.org/index.php/Pathway:WP5280), [WP5292](https://www.wikipathways.org/index.php/Pathway:WP5292) and add the Approved tag?
        - Incomplete pathway [WP5291](https://www.wikipathways.org/index.php/Pathway:WP5291)
- Search function works!

With kind regards,
Susan

@andrawaag , you are up next.

**Comment by DeniseSl22** [comment]
@susancoort : thanks for checking! No, the "Under construction" tags for these lipid related PWs need to stay, until we hear from the expert that the content is approved ;)
